src/server.c

```diff
@@ -38,7 +39,7 @@ int dqlite__init(struct dqlite_node *d,
 	registry__init(&d->registry, &d->config);
 	rv = uv_loop_init(&d->loop);
 	if (rv != 0) {
-		/* TODO: better error reporting */
+		snprintf(d->errmsg, DQLITE_ERRMSG_BUF_SIZE, "uv_loop_init(): failed");
```

Comment:
It'd be nice to include the error message from libuv, e.g.:

```C
snprintf(d->errmsg, DQLITE_ERRMSG_BUF_SIZE, "uv_loop_init(): %s", uv_strerror(rv));
```

See http://docs.libuv.org/en/v1.x/errors.html#c.uv_strerror